README.md

```diff
@@ -32,8 +19,6 @@ To get a local copy up and running follow these simple example steps.
 
 ### Usage
 
-### Run tests
-
 ### Deployment
```

Comment:
## OPTIONAL

- [ ] Please remove unused sections from your README to look more professional. 👍